how-to/operating-launchpad.rst

```diff
@@ -13,4 +13,5 @@ Operating Launchpad
    deploying-configuration-changes
    land-update-for-loggerhead
    transfer-project-ownership
-   create-bot-account
\ No newline at end of file
+   create-bot-account
+   porting-builders-to-newer-ubuntu-versions
```

Comment:
Maybe end a new line at the end to avoid this kind of strange diff when an additional line gets added.